### ISSUE TITLE
Cannot find some characters

### ISSUE THREAD

**Issue by merlinuwe** [BambusControl/obsidian-unicode-search#1]
Thank you for your great plugin.

I like to use 

Rightwards Harpoon Over Leftwards Harpoon

⇌ (U+21CC)

but I cannot find it.

Isn't it possible to find all unicode signs, yet?

I'm on android.

**Comment by BambusControl** [comment]
Hello @merlinuwe. You're right.

After taking a quick look, a lot of characters are missing from the list.
I will need to take deeper look into how to pull *all* Unicode characters.

Thanks for taking the time and effort to write the issue.

**Comment by BambusControl** [comment]
# Problem

Only a *few* pages of Unicode characters were initially put into the plugin.
That means a lot of **characters are missing**.

# Temporary Workaround

 **Edit the plugin data file.**

1. Close your Obsidian Vault
2. Open the `/.obsidian/plugins/unicode-search/data.json` file of your vault.
3. Go to the very end of the file.
4. Add a comma after the last record.
5. Append the info for a character you wish to add.

```json
{
    "meta": {
        "initialized": true,
        "version": "1"
    },
    "data": {
        //... a bunch of characters will be here ...
        "𝇩": {
            "name": "zorro",
            "char": "𝇩"
        }, // 3. Add a comma here
        // 4. [start] Add your character info here
        "⇌": {
            "name": "rightwards harpoon over leftwards harpoon",
            "char": "⇌"
        }
        // 4. [end]
    }
}
```

**Comment by BambusControl** [comment]
Fixed in **Release [0.4.0](https://github.com/BambusControl/obsidian-unicode-search/releases/tag/0.4.0)**.
However, not all characters are present, as it proved a strain on the search performance.

**Comment by ShahriarKh** [comment]
Thanks for this awesome plugin! Maybe add this guide in the README?
